### PR TITLE
Add myself as approver and reviewer

### PR DESCRIPTION
**Pull request by mads-hartmann**
## Description

To test https://github.com/gitpod-io/gitbot/pull/55 I have to create a PR. I thought I might as well add myself as a reviewer/approver as that would be useful going forward anyway.

## Related Issue(s)

Part of gitpod-io/gitpod#6623

## How to test

Someone other than me have to review the PR using the GitHub review status. Once approved tide should merge the PR.

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod-test-repo/pull/204#" title="Author self-approved">mads-hartmann</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **roboquat** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @roboquat` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=gitpod-io%2Fgitpod-test-repo).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod-test-repo/blob/1test/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by meysholdt — APPROVED** [review]

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: c517e8ea4c2f2d2fe8b6ab8180d626fbf4b9dc92</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod-test-repo/pull/204#" title="Author self-approved">mads-hartmann</a>*, *<a href="https://github.com/gitpod-io/gitpod-test-repo/pull/204#pullrequestreview-813407338" title="Approved">meysholdt</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **roboquat** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @roboquat` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=gitpod-io%2Fgitpod-test-repo).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod-test-repo/blob/1test/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
